src/Utilities/Tuple.hpp

```diff
@@ -174,3 +174,33 @@ constexpr inline void tuple_transform(
       tuple, std::forward<N_aryOp>(op),
       std::make_index_sequence<sizeof...(Elements)>{}, args...);
 }
+
+namespace tuple_impl_detail {
+
+template <size_t Start, typename... AllElements, size_t... Is>
+auto slice_impl(const std::tuple<AllElements...>& tuple,
+                std::index_sequence<Is...> /*meta*/) noexcept {
+  return std::make_tuple(std::get<Is + Start>(tuple)...);
+}
+
+}  // namespace tuple_impl_detail
+
+/// The subset of elements in `tuple` from index `Start` to (excluding) `Stop`
```

Comment:
These should be in the `UtilitiesGroup` otherwise I doubt anyone can find them in the documentation